<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMaskingWidgetBase</class>
 <widget class="QWidget" name="QgsMaskingWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QgsMessageBar" name="mMessageBar">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Masked symbol layers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsSymbolLayerSelectionWidget" name="mMaskTargetsWidget" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Masking symbol layers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsMaskSourceSelectionWidget" name="mMaskSourcesWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMaskSourceSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgsmasksourceselectionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsSymbolLayerSelectionWidget</class>
   <extends>QWidget</extends>
   <header>qgssymbollayerselectionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsMessageBar</class>
   <extends>QFrame</extends>
   <header>qgsmessagebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
